Essentials/Commands/CleanupModule.cs

```diff
@@ -19,6 +19,16 @@ public void Scan()
             Context.Respond($"Found {count} grids matching the given conditions.");
         }
 
+        [Command("list", "Lists grids matching the given conditions: hastype, notype, hassubtype, nosubtype, blockslessthan, blocksgreaterthan, ownedby")]
+        public void List()
+        {
+            var grids = ScanConditions(Context.Args).OrderBy(g => g.DisplayName).ToList();
+            for (var i = 0; i < grids.Count; i++)
+                Context.Respond($"{i + 1}. {grids[i].DisplayName} ({grids[i].BlocksCount} block(s))");
```

Comment:
Consider using a StringBuilder here and only send one chat message for the command instead of one message per grid.